nengo_gui/static/menu.js

```diff
@@ -27,6 +27,8 @@ Nengo.Menu.prototype.show = function (x, y, items) {
         return;
     }
 
+    this.menu_margin = 15;
+
```

Comment:
What browser is that?  On Chrome and Firefox for me it goes right to the edge.  Is it the same for you in both Chrome and Firefox?